Dockerfile.amd64

```diff
@@ -31,6 +31,38 @@ ADD k8s-install/scripts/calico.conf.default /calico.conf.tmp
 RUN microdnf install hostname && mkdir /licenses
 COPY LICENSE /licenses
 
+RUN microdnf update
+
+# Remove packages currently known to have vulnerabilities and dependencies
+# Remove packages deemed known to be unsafe by threat defense team
+RUN microdnf remove cyrus-sasl-lib \
+    openldap gnupg2 libcurl gpgme curl librepo libdnf rpm microdnf rpm-lib libsolv rpm-libs \
+    libdb-utils \
+    libdb \
+    bzip2-libs \
+    python3-libs platform-python libarchive crypto-policies platform-python-setuptools gnutls libssh glib2 openssl-libs \
+    krb5-libs librhsm libpeas libtirpc libmodulemd1 libnsl2 gobject-introspection json-glib \
+    expat \
+    libmetalink \
+    libarchive \
+    libssh-config \
+    libtasn1 \
+    p11-kit-trust ca-certificates \
+    openldap \
+    libgcrypt \
+    systemd-libs libusbx \
+    libidn2 \
+    libpsl \
+    libnghttp2 \
+    libcrypto  glibc-langpack-en glibc-minimal-langpack  json-c libcom_err \
+    libgcc libstdc++ libstdc libpcap libxml2 lz4-libs openssl platform-python-pip python3-pip-wheel systemd sqlite-libs \
+    libuuid \
+    libblkid libmount \
+    libsmartcols
+
+FROM scratch
```

Comment:
please comment on motivation, e.g.
```
# Copy everything into a fresh scratch image so that naive CVE scanners don't pick up binaries and libraries
# that have been removed in our later layers.
```